gnomad_qc/v3/annotations/generate_sv_hists.py

```diff
@@ -31,6 +33,8 @@ def generate_hists(mt: hl.MatrixTable) -> hl.Table:
     :return: Table with histograms.
     """
     logger.info("Generating histograms for age and GQ...")
+    # Remove multiallelic sites as recommended by the SV team
```

Comment:
sorry, I know I wrote the initial suggestion, but adding CNVs now also
```suggestion
    # Remove multiallelic CNV sites as recommended by the SV team
```